.github/dependabot.yaml

```diff
@@ -0,0 +1,30 @@
+version: 2
+updates:
+  -
+    package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
I think as a first step, I would go weekly to see how much updates are needed on a regular basis. 
Then based on  the amount of pullrequest you switch to monthly if needed

Reply:
I was wondering if we have the bandwidth, but yes let's do it!